src/fides/api/ops/schemas/privacy_notice.py

```diff
@@ -52,6 +52,40 @@ def validate_data_uses(self, valid_data_uses: List[str]) -> None:
             if data_use not in valid_data_uses:
                 raise ValueError(f"Unknown data_use '{data_use}'")
 
+    @root_validator
+    def validate_consent_mechanisms_and_display(
+        cls, values: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        """
+        Add some validation on where certain consent mechanisms must be displayed
+        """
+        consent_mechanism: Optional[str] = values.get("consent_mechanism")
+        displayed_in_overlay: Optional[bool] = values.get("displayed_in_overlay")
+        displayed_in_privacy_center: Optional[bool] = values.get(
+            "displayed_in_privacy_center"
+        )
```

Comment:
If users already have privacy notices saved that violate this, we would have failures.  In my opinion, users should just clear or update existing notices and notices histories while this feature is still in development